pkgs/applications/office/appflowy/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, gtk3, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "appflowy";
+  version = "0.0.2";
+
+  src = fetchzip {
+    url = "https://github.com/AppFlowy-IO/appflowy/releases/download/${version}/AppFlowy-linux-x86.tar.gz";
+    sha256 = "1fvv4mlgf0vqcq5zh0zl2xr44saz0sm47r8whcywwrmcm0l66iv6";
+  };
+
+  nativeBuildInputs = [
+      autoPatchelfHook
+      makeWrapper
+  ];
+
+  buildInputs = [
+      gtk3
+      openssl
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/opt/
+    mkdir -p $out/share/applications/
+    mkdir -p $out/bin/
+
+    # Copy archive contents to the outpout directory
+    cp -r ./* $out/opt/
+
+    # Create the desktop file
+    echo -e "[Desktop Entry] \nName=AppFlowy \nComment=An Open Source Alternative to Notion \nIcon=$out/opt/flowy_logo.svg \nExec=$out/bin/appflowy \nCategories=Office \nType=Application \nTerminal=false" > $out/share/applications/appflowy.desktop
+
+    # Add missing libraries to appflowy using the ones it comes with
+    makeWrapper $out/opt/app_flowy $out/opt/app_flowy_warpped --set LD_LIBRARY_PATH "$out/opt/lib/"
+
+    # Symlink wrapper
+    ln -s $out/opt/app_flowy_warpped $out/bin/appflowy
```

```suggestion
    # Add missing libraries to appflowy using the ones it comes with
    makeWrapper $out/opt/app_flowy $out/bin/appflowy \
      --set LD_LIBRARY_PATH "$out/opt/lib/"
```
Why do we need to link it? Can't we just immediately write the wrapper to the location?

---

Why not ? I will try this